pkg/manager/impl/execution_manager.go

```diff
@@ -263,6 +265,41 @@ func assignResourcesIfUnset(ctx context.Context, identifier *core.Identifier,
 	return resourceEntries
 }
 
+func resolveTaskLimitsAndPlatformRequestDefaults(ctx context.Context, identifier *core.Identifier,
```

Comment:
I don't see the code updating the limits (as the name suggests?) am I missing something?

Reply:
nope, I just struggled with naming. is 'considerTaskLimitsAndPlatformRequestDefaults` or `reconcileTaskLimitsAndPlatformRequestDefaults` any better? other suggestions welcomed

Reply:
the limit is not supposed to change though... the request is.

Reply:
okay updated